packages/speech/src/index.js

```diff
@@ -0,0 +1,919 @@
+ * speech.createDetectJob(file, {
+ *   encoding: 'LINEAR16',
+ *   sampleRate: 16000
+ * }, function(err, operation, apiResponse) {});
+ *
+ * //-
+ * // <h3>Run speech detection over a file in Cloud Storage</h3>
+ * //-
+ * var file = 'gs://your-bucket-name/bridge.raw';
+ * speech.createDetectJob(file, {
+ *   encoding: 'LINEAR16',
+ *   sampleRate: 16000
+ * }, function(err, operation, apiResponse) {
+ *   operation.on('complete', function(err, operation, apiResponse) {
+ *     // operation = {
+ *     //   response: {
+ *     //     results: [
+ *     //       {
+ *     //         alternatives: [
+ *     //           {
+ *     //             transcript: "...",
+ *     //             confidence: 0.88
+ *     //           }
+ *     //         ]
+ *     //       }
+ *     //     ]
+ *     //   }
+ *     // }
+ *   });
+ * });
+ */
+Speech.prototype.createDetectJob = function(file, config, callback) {
+  var self = this;
+
+  Speech.findFiles_(file, function(err, files) {
+    if (err) {
+      callback(err);
+      return;
+    }
+
+    var reqOpts = {
+      config: extend({
+        encoding: Speech.detectEncoding_(file)
+      }, config),
+      audio: files[0]
+    };
+
+    self.createRecognizeJob(reqOpts, callback);
+  });
+};
+
+/**
+ * Perform bidirectional streaming speech-recognition: receive results while
+ * sending audio.
+ *
+ * This is an advanced API method that requires raw
+ * [`StreamingRecognizeRequest`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognizeRequest)
+ * objects. If that doesn't sound like what you're looking for, you'll probably
+ * appreciate {module:speech#streamingDetect}.
+ *
+ * @resource [`StreamingRecognize` API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.Speech.StreamingRecognize}
+ * @resource [`StreamingRecognizeRequest` API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognizeRequest}
+ *
+ * @param {object} streamingRecognizeRequest - A `StreamingRecognizeRequest`
+ *     object. See [`StreamingRecognizeRequest`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognizeRequest).
+ * @param {object} streamingRecognizeRequest.streamingConfig - A
+ *     `StreamingRecognitionConfig` object. See [`StreamingRecognitionConfig`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig).
+ * @param {object} streamingRecognizeRequest.streamingConfig.config - A
+ *     `RecognitionConfig` object. See [`RecognitionConfig`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig).
+ * @param {string} streamingRecognizeRequest.streamingConfig.config.encoding -
```

Comment:
`dox` flips out if the beginning of the description is not on the same line as the name.

e.g. http://callmehiphop.github.io/gcloud-node/#/docs/speech/master/speech?method=createRecognizeStream

If you've rebased master, I made a small adjustment to allow comments to break the 80 character rule.

Also in other APIs we've always called this argument (streamingReconigzeRequest) `options` which would shorten the character count here.